docs/dictionary/control_st/if.lcdoc

```diff
@@ -4,9 +4,30 @@ Type: control structure
 
 Syntax: if <condition> then <statementList> [else <elseStatementList>]
 
+Syntax:if <condition> then
+   <statementList>
+ [else if <condition> then
+   <elseifStatementList>]
+ [else 
+   <elseStatementList>]
+end if
+
+
 Summary:
-<execute|Executes> a list of <statement|statements> if a condition is
-true. 
+Evaluates a <conditional expression|condition> to determine if it is true
+and if so <execute|executes> a subsequent <statment|statement> or <statementlist|statement list>.
```

Comment:
this should just be `<statement>` as `<A|A>` is equivalent to `<A>` (note the LHS here has a typo, missing an 'e')